go.mod

```diff
@@ -5,5 +5,6 @@ go 1.14
 require (
 	github.com/google/go-cmp v0.5.2
 	golang.org/x/tools v0.0.0-20200416214402-fc959738d646
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
This is used in the test for `SetConfigBytes` (specifically, it provides the `yaml.Marshal` function).

Reply:
The tests appear to pass if you use `sigs.k8s.io/yaml` in the test rather than the `gopkg`.

Reply:
You are correct. Not sure what happened here. Removed.